cmd/dvol-docker-plugin/dvol-docker-plugin.go

```diff
@@ -106,6 +106,14 @@ func main() {
 				writeResponseErr(fmt.Errorf("Could not create volume %s: %v", name, err), w)
 				return
 			}
+
+			// XXX: This is copied from the code for `dvol init`. Reuse that code.
+			// XXX: There's already a constant for default branch. Use that.
```

Comment:
Any reason why you don't action these right now? Both seem like small jobs, especially the constant.